README.md

```diff
@@ -61,6 +61,7 @@ Explore the following documentation to find out which tools can help you in your
 	- [`tcpconnect`](docs/gadgets/trace/tcpconnect.md)
 	- [`tcpdrop`](docs/gadgets/trace/tcpdrop.md)
 	- [`tcpretrans`](docs/gadgets/trace/tcpretrans.md)
+- [`prometheus`](docs/gadgets/prometheus.md)
```

Comment:
Need to update ["How to use"](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/README.md#how-to-use) section as well!

Reply:
oh thanks. I was assuming it was a flake test given that it's run in the integration tests.